dashboard/app/controllers/pd/workshop_material_orders_controller.rb

```diff
@@ -31,10 +31,12 @@ def create
   # GET /pd/workshop_materials
   def admin_index
     @workshop_material_orders = Pd::WorkshopMaterialOrder.order(created_at: :desc)
-    @workshop_material_orders.find_each do |order|
-      order.refresh
-      order.save!
-    end
+    @stats = {
+      total_count: @workshop_material_orders.count,
+      succesfully_ordered: @workshop_material_orders.successfully_ordered.count,
+      shipped: @workshop_material_orders.shipped.count,
+      order_errors: @workshop_material_orders.with_order_errors.count
+    }
```

Comment:
nit - the key names should say count. You aren't returning the errors or shipped orders, you're returning the count of them